index.php

```diff
@@ -0,0 +1,5 @@
+<?php
+
+require './controller/controller.php';
```

Comment:
Why you don't add composer autoloader to have access to libs ?

require 'vendor/autoload.php';